netmanager/src/config/urls/faultDetection.js

```diff
@@ -0,0 +1,7 @@
+import { stripTrailingSlash } from '../utils';
+
+const BASE_FAULT_DETECTION_URL = stripTrailingSlash(
+  process.env.BASE_FAULT_DETECTION_URL || process.env.REACT_APP_BASE_URL
```

Comment:
Hi @Codebmk 
React environment variables **MUST** always start with `REACT_APP_` otherwise they will not be rendered into the website runtime

Reply:
Kindly change `BASE_FAULT_DETECTION_URL` to something like `REACT_APP_BASE_FAULT_DETECTION_URL` instead